WGLMakie/src/three_plot.jl

```diff
@@ -4,6 +4,14 @@ end
 
 JSServe.session(td::ThreeDisplay) = td.session
 
+function GeometryBasics.widths(screen::ThreeDisplay)
+    # look at d.qs().clientWidth for displayed width
+    width = Int(WGLMakie.JSServe.evaljs_value(screen.session, WGLMakie.JSServe.js"document.querySelector('canvas').width"; time_out=100))
```

Comment:
Every evaljs_value call is super expensive, can we make this one call? Have you tried if this works?

Reply:
Will try that - was unsure how the structure of jsserve worked but will try to return an array or something.

Reply:
Looked like it worked - thanks for the suggestion!  Should be good to go from my end.